service/history/history_engine_test.go

```diff
@@ -424,41 +425,37 @@ func (s *engineSuite) TestGetMutableStateLongPoll_CurrentBranchChanged() {
 	asyncBranchTokenUpdate := func(delay time.Duration) {
 		timer := time.NewTimer(delay)
 		<-timer.C
-		newExecution := &commonpb.WorkflowExecution{
-			WorkflowId: execution.WorkflowId,
-			RunId:      execution.RunId,
-		}
-		ms.GetExecutionInfo().GetVersionHistories()
 		s.mockHistoryEngine.eventNotifier.NotifyNewHistoryEvent(events.NewNotification(
-			"tests.NamespaceID",
-			newExecution,
+			ms.GetWorkflowKey().NamespaceID,
```

Comment:
@yux0 I see you authored this test. Do you still remember what this test is supposed to test? 
It's called CurrentBranchChanged but I don't see any logic related to branch change and test is asserting two current branch token are equal.